app/test/std_sdk_test.go

```diff
@@ -319,22 +305,15 @@ func getAddress(account string, kr keyring.Keyring) sdk.AccAddress {
 		panic(err)
 	}
 	return addr
-=======
+}
+
 func (s *StandardSDKIntegrationTestSuite) TestGRPCQueries() {
 	t := s.T()
-	t.Run("testnode can query network min gas price", func(t *testing.T) {
-		queryClient := minfee.NewQueryClient(s.cctx.GRPCClient)
-		resp, err := queryClient.NetworkMinGasPrice(s.cctx.GoContext(), &minfee.QueryNetworkMinGasPrice{})
-		require.NoError(t, err)
-		got, err := resp.NetworkMinGasPrice.Float64()
-		require.NoError(t, err)
-		assert.Equal(t, v2.NetworkMinGasPrice, got)
-	})
 	t.Run("testnode can query local min gas price", func(t *testing.T) {
 		serviceClient := nodeservice.NewServiceClient(s.cctx.GRPCClient)
 		resp, err := serviceClient.Config(s.cctx.GoContext(), &nodeservice.ConfigRequest{})
 		require.NoError(t, err)
-		want := "0.002000000000000000utia"
+		want := ""
```

Comment:
[question] why does the min gas price go from `.002` to `""` in this PR?

Reply:
maybe the testnode didn't configure a local min gas price

Reply:
Yeah it looks like testnode doesn't set it. I think we may have made it set the min gas price in v2 when we were doing the minimum gas price work. I didn't want to touch it because it's orthogonal to this PR. If you think we should set it like we do in v2 (and onwards), then we can make a separate issue for it.

Reply:
I think what you've done is fine, it just seemed worth asking about because it diverged from the PR that backports this to v2.x so I wasn't sure if this was intentional.